build/env.default

```diff
@@ -0,0 +1,6 @@
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_SECURITY_TOKEN
+S3_BUCKET
```

Comment:
Later on in the dockerfile we're re-defining these values

```
ENV S3_BUCKET=heroku-activestorage-default
ENV S3_PREFIX=dist_heroku-20_amd64/
ENV S3_REGION=s3.us-east-1
```

I think we can/should remove the `S3_` prefixed env vars here. Either that or remove them from the dockerfiles, but set them here.

Reply:
This is intentional. The defaults in `Dockerfile` are so that you don't have to supply your own values unless you're e.g. forking this buildpack and wanna host your own build.

The empty values in `env.default` tell Docker to fetch env vars of that name from the calling environment.

So when using the image e.g. in a GH Codespace, you set three Codespace secrets (the three `AWS_…` values) which are exposed as env vars, and then you can simply do `docker run … --env-file=env.default …`, and it'll take the three `AWS_…` values from the environment, and the three `S3_…` values from the defaults in `Dockerfile`.